app/src/main/java/com/kamron/pogoiv/IVScanResult.java

```diff
@@ -138,19 +138,19 @@ public void addIVCombination(int attackIV, int defenseIV, int staminaIV) {
 
     /**
      * Checks if the previous scanned pokemon can be the same pokemon as the one scanned 2 scans ago
-     * checks if newer scan has same or higher level, and same or better evolution. (because pokemon cant de-level or devolve)
+     * checks if newer scan has higher level, and same or next evolution. (because pokemon cant de-level or devolve)
```

Comment:
I'd say you can refine a pokemon if you *either* powered it up *or* evolved it. Unless you're sure that evolving it can never lead to refinements.